### PR TITLE
feat: Enable nullable

### PR DESCRIPTION
**Pull request by HavenDV**

### CODE REVIEW

File: MimeMapping.sln [hunk 1]
```diff
@@ -6,7 +6,6 @@ MinimumVisualStudioVersion = 10.0.40219.1
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{53C2C92A-AA91-4027-A6A3-AAECEE661B38}"
 	ProjectSection(SolutionItems) = preProject
 		.editorconfig = .editorconfig
-		MimeMapping.nuspec = MimeMapping.nuspec
```
Comment (viceice) [inline MimeMapping.sln:9]: Why this change?

Comment (HavenDV) [reply MimeMapping.sln:9]: You forgot to delete this from .sln, it doesn't exist

**Review by viceice — APPROVED** [review]
